docs/index.html

```diff
@@ -75,7 +75,7 @@
         ctx     = canvas.getContext('2d'),
         ucanvas = get('upcoming'),
         uctx    = ucanvas.getContext('2d'),
-        speed   = { start: 1000.6, decrement: 0.005, min: 1.1 }, // how long before piece drops by 1 row (seconds)
+        speed = { start: 0.6, decrement: 0.005, min: 0.1 }, // how long before piece drops by 1 row (seconds)
```

Comment:
Gut - aber ich würde `0.006`nehmen: 

```suggestion
        speed = { start: 0.6, decrement: 0.006, min: 0.1 }, // how long before piece drops by 1 row (seconds)
```